### DISCUSSION TITLE
How to use mitmweb to rewrite/modify response body(json)?

### DISCUSSION THREAD

**Discussion by passcodex** [mitmproxy/mitmproxy#4962] [Q&A] [answered]
I want to modify "/unifiedauthapi/" response body,

"id":72 > "id":75
![image](https://user-images.githubusercontent.com/11014056/143556612-b7581bb8-062c-48b1-8030-2fcdf912f9dc.png)

The command line of mitmweb is 
`/Users/user/mitmproxy/mitmweb --web-host 0.0.0.0 \
--modify-body "|~m GET & ~u /unifiedauthapi/|"id":72|"id":75"
 
but it's not working.

How should I use mitmweb to make it work?

**Comment by mhils** [comment, marked as answer]
This looks correct. Could it be that there is whitespace between `"id":` and `72`?

**Reply by passcodex** [reply]
> This looks correct. Could it be that there is whitespace between `"id":` and `72`?
Thanks!
I need to add back slash \ in front of special string.
`"|~m GET & ~u /unifiedauthapi/|\"id\":\\s*\\S*72|\"id\":75"`
